src/Nix/TH.hs

```diff
@@ -69,7 +69,8 @@ freeVars e = case unFix e of
     staticKey (DynamicKey _) = Nothing
 
     bindDefs :: Binding r -> Set VarName
-    bindDefs (Inherit _ keys _) = Set.fromList $ mapMaybe staticKey keys
+    bindDefs (Inherit Nothing _ _) = Set.empty;
```

Comment:
Technically this aren't the binding definitions anymore, because `inherit x;` does define `x`, but that isn't returned anymore. Maybe it should be named `killSet` instead, from https://en.wikipedia.org/wiki/Reaching_definition

Reply:
I don't think it matters too much, seeing it is an internal function that is not exported anywhere. Good names are good though!